stats.go

```diff
@@ -335,44 +331,37 @@ func (e *statsExporter) viewToMetricDescriptor(ctx context.Context, v *view.View
 // An error will be returned if there is already a metric descriptor created with the same name
 // but it has a different aggregation or keys.
 func (e *statsExporter) createMeasure(ctx context.Context, v *view.View) error {
-	e.createdViewsMu.Lock()
-	defer e.createdViewsMu.Unlock()
+	// Skip create metric descriptor if configured
+	if e.o.SkipCMD {
+		return nil
+	}
+
+	e.metricMu.Lock()
+	defer e.metricMu.Unlock()
 
 	viewName := v.Name
 
-	if md, ok := e.createdViews[viewName]; ok {
-		// [TODO:rghetia] Temporary fix for https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/issues/76#issuecomment-459459091
-		if builtinMetric(md.Type) {
-			return nil
-		}
-		return e.equalMeasureAggTagKeys(md, v.Measure, v.Aggregation, v.TagKeys)
```

Comment:
is this check not required anymore?

Reply:
This is not used in metrics or metrics_proto. I think we tried to do too much in the client. If the metric changed the definition user should deal with this by changing the name. We cannot correct this error.

Reply:
SG